Gems/ROS2/Code/Source/RobotImporter/URDF/VisualsMaker.cpp

```diff
@@ -231,7 +232,54 @@ namespace ROS2
         entity->Deactivate();
     }
 
-    static void OverrideMaterialPbrSettings(const sdf::Material* material, [[maybe_unused]] AZ::Render::MaterialAssignmentMap& overrides)
+    static void OverrideScriptMaterial(const sdf::Material* material, [[maybe_unused]] AZ::Render::MaterialAssignmentMap& overrides)
```

Comment:
The `overrides` parameter is no longer unused.
```suggestion
    static void OverrideScriptMaterial(const sdf::Material* material, AZ::Render::MaterialAssignmentMap& overrides)
```